scriptmodules/supplementary/splashscreen/asplashscreen.sh

```diff
@@ -8,22 +8,14 @@ REGEX_IMAGE=""
 # Load user settings
 . /opt/retropie/configs/all/splashscreen.cfg
 
-is_fkms() {
-    if grep -q okay /proc/device-tree/soc/v3d@7ec00000/status 2> /dev/null || grep -q okay /proc/device-tree/soc/firmwarekms@7e600000/status 2> /dev/null ; then
-        return 0
-    else
-        return 1
-    fi
-}
-
 do_start () {
     local config="/etc/splashscreen.list"
     local line
     local re="$REGEX_VIDEO\|$REGEX_IMAGE"
-    local omxiv="/opt/retropie/supplementary/omxiv/omxiv"
+    local vlc="vlc --intf dummy --quiet --no-video-title-show --play-and-exit"
     case "$RANDOMIZE" in
         disabled)
-            line="$(head -1 "$config")"
+            line="$(cat "$config" | xargs)"
```

Comment:
Thanks.

Reply:
This will need some additional changes - the idea was to pass all the files in `/etc/splashscreen.list` to vlc if RANDOMIZE is disabled. but then line is passed to vlc quoted and I don't think this will work (And may not have before too).

Reply:
This should be working now.